packages/babel-plugin-proposal-decorators/test/fixtures/2021-12-metadata--to-es2015/getting-previously-set-metadata/exec.js

```diff
@@ -13,7 +13,7 @@ function dec2(_, { setMetadata, getMetadata }) {
 }
 
 class Foo {
-  @dec1 @dec2 a;
+  @dec2 @dec1 a;
```

Comment:
In this test we check whether `desc2` should see the metadata changed by `desc1`. Because decorators are applied from right to left, the order should be swapped.